lib/taiko.js

```diff
@@ -112,7 +113,9 @@ module.exports.openBrowser = async (options = { headless: true }) => {
     currentHost = endpoint.host;
     currentPort = endpoint.port;
     await connect_to_cri();
-    return { description: 'Browser opened' };
+    const device = commandlineArgs().emulateDevice;
```

Comment:
The command line logic can go to the runner, and can be accessed as Env variable ( TAIKO_EMULATE_DEVICE )from API.

Reply:
I have address as per @zabil comment https://github.com/getgauge/taiko/pull/340#issuecomment-454652434, so should i remove from commandLineArg and just add ENV access ?

Reply:
we will need both `taiko script.js --emulateDevice=iPhone` and `TAIKO_EMULATE_DEVICE=iphone gauge run specs` when using other runners

Reply:
Done